engine/src/main/scala/cromwell/engine/workflow/lifecycle/finalization/CopyWorkflowOutputsActor.scala

```diff
@@ -97,6 +101,21 @@ class CopyWorkflowOutputsActor(workflowId: WorkflowId,
     Future.sequence(copies)
   }
 
+  private def moveWorkflowOutputs(workflowOutputsFilePath: String): Future[Seq[Unit]] = {
+    val workflowOutputsPath = buildPath(workflowOutputsFilePath)
+    val outputFilePaths = getOutputFilePaths(workflowOutputsPath)
+
+    markDuplicates(outputFilePaths)
+
+    val moves = outputFilePaths map { case (srcPath, dstPath) =>
+      asyncIo.copyAsync(srcPath, dstPath) flatMap { _ =>
+        asyncIo.deleteAsync(srcPath)
```

Comment:
Do we have anything in the tests anywhere that validates that we only delete the file when the copy is successful?

Reply:
Added `gcpWdlResultsMovingFail` case.